src/common/Spine.ts

```diff
@@ -169,7 +181,7 @@ export class Spine<N extends IR.Node = IR.Node> {
         }
       }
       return curr;
-    } else if (skipReplaced) {
+    } else if (skipReplacement || skipChildren) {
       return this.replacedWith(ret);
```

Comment:
Looks like `skipReplaced` (function parameter) is ignored now.

Reply:
It's not ignored, it's the default (initial) value of `skipReplacement`. We want to keep the old logic for skipping but additionally skip when requested from plugin. But I cannot merge the two variables, because I need to pass the original value from function parameter to the recursive calls.